recipes/openmm-cpu/conda_build_config.yaml

```diff
@@ -0,0 +1,3 @@
+python:
```

Comment:
you do not need this file, you just have to select those pythons which you want to in your recipe

Reply:
Thanks for the suggestion! Applied!